test/test-pipe-set-fchmod.c

```diff
@@ -0,0 +1,30 @@
+#include "uv.h"
+#include "task.h"
+
+TEST_IMPL(pipe_set_fchmod) {
+  uv_pipe_t pipe_handle;
+  uv_loop_t* loop;
+  int r;
+
+  loop = uv_default_loop();
+
+  r = uv_pipe_init(loop, &pipe_handle, 0);
+  ASSERT(r == 0);
+
+  r = uv_pipe_bind(&pipe_handle, TEST_PIPENAME);
+  ASSERT(r == 0);
+
+  /* No easy way to test if this works, we will only make sure that */
+  /* the call is successful. */
+  r = uv_pipe_chmod(&pipe_handle, UV_READABLE);
```

Comment:
Under *nix I do [fchmod](https://github.com/libuv/libuv/pull/1386/files#diff-1dcc314fa3f88181e7dcad56df613e2bR330). Should it be something else?